TWLight/applications/views.py

```diff
@@ -1076,7 +1076,7 @@ def post(self, request, *args, **kwargs):
                 self.request,
                 messages.ERROR,
                 # Translators: this message is shown to coordinators who attempt to change an application's Status from INVALID to any other Status.
-                _("Status of INVALID applications Cannot be changed."),
+                _("Status of INVALID applications cannot be changed."),
```

Comment:
This is actually the only change you need to make - the .po files are all generated automatically when the string update is deployed :)

Reply:
(Which is why there are now merge conflicts)